gitgud/operations.py

```diff
@@ -34,6 +36,7 @@ def add_and_commit(self, name):
         # TODO Commits with the same time have arbitrary order when using git log, set time of commit to fix
         self.add_file_to_index(name)
         commit = self.repo.index.commit(name, author=actor, committer=actor, skip_hooks=True)
+        track_commit(self, name, commit.hexsha[:7])
```

Comment:
Why do we have this here, isn't this out of scope for this PR?

Reply:
It's there because all features (abort_commit, track_merges, track_rebases) are based off the track_commits branch. There are certain changes (like the hooks/init file, hooks paths, and __main__ formatting) that need to propagate to all feature branches. Whenever these need changing, they are done to the track_commits branch and then the other branches are rebased on top of track_commits. This comes with the side effect that track_commits stuff gets added, but most of that is pretty useful for debugging so I didn't mind it propagating.